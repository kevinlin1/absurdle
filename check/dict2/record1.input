argh
beta
flew
crew
else
