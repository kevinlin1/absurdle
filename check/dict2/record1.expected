9 words
> argh
: ⬜⬜⬜⬜

4 words
> beta
: ⬜🟨⬜⬜

2 words
> flew
: ⬜🟩🟨⬜

1 words
> crew
: ⬜⬜🟨⬜

1 words
> else
: 🟩🟩🟩🟩

Absurdle 5/∞

⬜⬜⬜⬜
⬜🟨⬜⬜
⬜🟩🟨⬜
⬜⬜🟨⬜
🟩🟩🟩🟩
